### PR TITLE
Update the changelog (commits after release 1.2.3)

### PR DESCRIPTION
**Pull request by Chocobozzz**
I made a draft of the changelog after the release 1.2.3 to help users to see what modifications were made and to help authors when they'll want to release a new version of Clementine.

My english is not perfect, I certainly missed some things and maybe some other features should be in major feature (or the opposite) so feel free to correct it :)

Note: I didn't include commits for the Clementine remote application.

### CODE REVIEW

File: Changelog [hunk 1]
```diff
@@ -1,3 +1,101 @@
...
+    * Improve the organize dialog
+    * Add playlist save preference
+    * Add a preference to disable the pause notification
+    * Add a preference tab to hide some internet services
+    * Add an option to disable inline song metadata editing
+    * Use a save dialog option instead of quick change menu
+    * Add ability to fetch lyrics from lololyrics.com
+    * Add support for monitors in portrait mode
+    * Add now playing widget mode
+    * Add icons to extra
```
Comment (Fitoschido) [inline Changelog]: What does this mean for a casual user? =)

Comment (Chocobozzz) [inline Changelog]: In the extra tab, icons were added (e.g : all glory for the Hypnotoad)

File: Changelog [hunk 2]
```diff
@@ -1,3 +1,101 @@
+Version 1.3: (stopped at commit a61eac109c49bd0abfa0325b9da35ac60039921d)
```
Comment (hatstand) [inline Changelog]: Remove the commit info

**Comment by Chocobozzz** [comment]
Done :)

**Pull request merged**